tailor.yml

```diff
@@ -0,0 +1,25 @@
+---
+serge:
+  config: .serge
+ts:
+  kind: crowdin
+  config: crowdin.yml
+  name: "Extension: Domain Connect"
+  identifier: ext-domain-connect
+  join_policy: private
+  language_access_policy: moderate
+  language_mapping:
+    es: es-es
+    sv: sv-se
+    pt: pt-pt
+    tl-ph: fil
+trigger:
+  kind: timer
+  schedule: "20 1,13 * * 1-5"
```

Comment:
Is it necessary to run the CL pipeline exactly at 1:20 and 13:20 UTC ? In order to avoid the system load peaks it will be better to run the CL pipeline for different projects at different time.

Reply:
moved a half of hour later